src/util/quicktune.h

```diff
@@ -65,19 +67,15 @@ void setQuicktuneValue(const std::string &name, const QuicktuneValue &val);
 
 void updateQuicktuneValue(const std::string &name, QuicktuneValue &val);
 
-#ifndef NDEBUG
-	#define QUICKTUNE(type_, var, min_, max_, name){\
-		QuicktuneValue qv;\
-		qv.type = type_;\
-		qv.value_##type_.current = var;\
-		qv.value_##type_.min = min_;\
-		qv.value_##type_.max = max_;\
-		updateQuicktuneValue(name, qv);\
-		var = qv.value_##type_.current;\
-	}
-#else // NDEBUG
-	#define QUICKTUNE(type, var, min_, max_, name){}
-#endif
+#define QUICKTUNE(type_, var, min_, max_, name) do { \
```

Comment:
Did you mean to turn this on with this PR?

Reply:
yes. the idea is to not restrict a debug feature to a debug build.